modules/client/src/internal/client/methods.ts

```diff
@@ -74,20 +76,33 @@ export class ClientMethods extends ClientCore implements IClientMethods {
    * @memberof ClientMethods
    */
   public async *create(
-    params: ICreateParams,
+    params: ICreateParams
   ): AsyncGenerator<DaoCreationStepValue> {
     const signer = this.web3.getConnectedSigner();
     if (!signer) {
-      throw new Error("A signer is needed");
+      throw new NoSignerError();
     } else if (!signer.provider) {
-      throw new Error("A web3 provider is needed");
+      throw new NoProviderError();
     }
 
     const daoFactoryInstance = DAOFactory__factory.connect(
       this.web3.getDaoFactoryAddress(),
-      signer,
+      signer
     );
 
+    const pluginInstallationData: DAOFactory.PluginSettingsStruct[] = [];
+    for (const plugin of params.plugins) {
+      const latestVersion = await PluginRepo__factory.connect(
+        plugin.id,
+        signer
+      ).getLatestVersion();
+      pluginInstallationData.push({
+        pluginSetup: latestVersion[1],
+        pluginSetupRepo: plugin.id,
+        data: plugin.data,
+      });
+    }
```

Comment:
We had `...unwrapCreateDaoParams(params)` doing this work in the code commented below. Why don't we use what we already had and keep things simpler here?